config/openvpn-client-export/openvpn-client-export.inc

```diff
@@ -807,7 +816,7 @@ function openvpn_client_export_sharedkey_config($srvid, $useaddr, $proxy, $zipco
 			}
 			$conf .= "http-proxy {$proxy['ip']} {$proxy['port']} ";
 		}
-		if ($proxy['proxy_type'] == "socks") 
+		if ($proxy['proxy_type'] == "socks")
```

Comment:
This seems to be caused by atom/vi stripping blank spaces at the end of the lines. Since this actually improves the code (there are obviously no extra spaces on any other lines, else they would have suffered the same treatment), I suggest letting these minor corrections through as part of the pull.